mailstore-connector/config/variables.yaml

```diff
@@ -26,4 +26,26 @@ Variables:
       properties:
           mail.imaps.ssl.checkserveridentity: false
           mail.imaps.ssl.trust: '*'
-  
\ No newline at end of file
+          # only set below credential when you go with oauth2
+          # mail.imaps.auth.mechanisms: 'XOAUTH2'
+          # mail.imaps.sasl.enable: 'true'
+          # mail.imaps.sasl.mechanisms: 'XOAUTH2'
+      
+      # Basic: username and password, AzureOauth2UserPasswordProvider: currently only support OAuth2 client credentials grant flow
+      # com.axonivy.connector.oauth.BasicUserPasswordProvider for Basic Authentication
+      # com.axonivy.connector.oauth.AzureOauth2UserPasswordProvider for AzureOauth2UserPasswordProvider
+      userPasswordProvider: 'com.axonivy.connector.oauth.BasicUserPasswordProvider'
+      
+      # [basic, oauth2] basic: username and password, oauth2: currently only support OAuth2 client credentials grant flow      
```

Comment:
I would not add here the specific settings for the Basic UserPasswordProvider, not for Azure.